lib/libvarnish/vsha256.c

```diff
@@ -33,8 +33,16 @@
 
 #ifndef __DARWIN_BYTE_ORDER
 #  include <endian.h>
-#  define VBYTE_ORDER	__BYTE_ORDER
-#  define VBIG_ENDIAN	__BIG_ENDIAN
+#  ifdef _BYTE_ORDER
```

Comment:
is there a particular reason we should check for `_BYTE_ORDER` rather than `__BYTE_ORDER` to be define?

Reply:
Because on some systems migth be no `__BYTE_ORDER` and only `_BYTE_ORDER`.

Reply:
it's probably not important, but what I meant is: wouldn't we achieve the same with

```c
# ifdef __BYTE_ORDER
#  define VBYTE_ORDER	__BYTE_ORDER
# else
#  define VBYTE_ORDER	_BYTE_ORDER
# endif
```

I am asking because we preferred `__BYTE_ORDER` before, so if you suggest to prefer `_BYTE_ORDER` now, I wanted to know if you had a particular reason?